core/sys/linux/sys.odin

```diff
@@ -2803,8 +2816,8 @@ getrandom :: proc "contextless" (buf: []u8, flags: Get_Random_Flags) -> (int, Er
 	Execute program relative to a directory file descriptor.
 	Available since Linux 3.19.
 */
-execveat :: proc "contextless" (dirfd: Fd, name: cstring, argv: [^]cstring, envp: [^]cstring) -> (Errno) {
-	ret := syscall(SYS_execveat, dirfd, cast(rawptr) name, cast(rawptr) argv, cast(rawptr) envp)
+execveat :: proc "contextless" (dirfd: Fd, name: cstring, argv: [^]cstring, envp: [^]cstring, flags: i32) -> (Errno) {
```

Comment:
Since you're adding flags could you make this value a bitset instead of an `i32`? It would help people interfacing with this syscall.

![image](https://github.com/odin-lang/Odin/assets/27109464/c8bdd62e-edad-4347-9f21-5aa29ba84621)